runtime/lua/bugreport.lua

```diff
@@ -0,0 +1,73 @@
+-- Copied and modified gist.github.com/soulik/82e9d02a818ce12498d1
+local function getOS()
+	local raw_os_name = ""
+
+	-- LuaJIT shortcut
+	if jit and jit.os then
+		raw_os_name = jit.os
+	else
+		-- is popen supported?
+		local popen_status, popen_result = pcall(io.popen, "")
```

Comment:
You could check windows using `vim.fn.has("win32") ~= 0`

Reply:
I think I changed it like you meant, but I honestly have no idea what I'm doing.